cmlutils/projects.py

```diff
@@ -1721,3 +1723,23 @@ def collect_import_application_list(self, project_id):
         self.metrics_data["total_application"] = len(app_name_list)
         self.metrics_data["application_name_list"] = sorted(app_name_list)
         return app_metadata_list, sorted(app_name_list)
+
+    def transfer_ownership(self, proj_id: str, target_owner: str):
+            endpoint = Template(ApiV2Endpoints.PATCH_PROJECT.value).substitute(
+                project_id=proj_id
+            )
+            json_data = {
+                "owner": {
+                    "username": target_owner
+                }
+            }
+            logging.info("Transfering the ownership of project to: %s", target_owner)
+            response = call_api_v2(
+                    host=self.host,
+                    endpoint=endpoint,
+                    method="PATCH",
+                    user_token=self.apiv2_key,
+                    json_data=json_data,
+                    ca_path=self.ca_path,
+                )
```

Comment:
Also, if anything fails in between - can it be logged

Reply:
If the API call fails error is printed inside "call_api_v2" function. I have tested this scenario as well.

Reply:
does it print endpoint too ?

Reply:
Yes